src/auth/guard/rootAdmin.guard.ts

```diff
@@ -0,0 +1,26 @@
+import {
```

Comment:
Not sure why you need this guard since you has a better one that can be used as a decorator with speified roles.  So you can use it as @Roles([Role.ROOT_ADMIN])